### PR TITLE
Title of the lp changed - I kindly ask for the review of the changes made

### PR DESCRIPTION
**Pull request by maciekpollub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="description"
           content="Application is supposed to help the user manage their free time in a not usuall manner.">
     <meta name="author" content="Red Coders Team">
-    <title>Yawn Killer</title>
```
Comment (HKrzyszczak) [inline index.html:9]: Old tittle was better.

**Review by HKrzyszczak — CHANGES_REQUESTED** [review]
Please change.

File: index.html (hunk 1 above)
Comment (maciekpollub) [reply index.html:9]: Ok.

**Review by HKrzyszczak — APPROVED** [review]

**Review by patrykomiotek — COMMENTED** [review]
:trollface:

**Pull request merged**